toolkit/tools/imagecustomizer/README.md

```diff
@@ -1,11 +1,11 @@
-# Mariner Image Customizer
+# Azure Linux Image Customizer
 
-The Mariner Image Customizer (MIC) is a tool that can take an existing generic Mariner
+The Azure Linux Image Customizer (MIC) is a tool that can take an existing generic Azure Linux
```

Comment:
Check with Chris Gunn on what he wants to do here.

Reply:
Checked with Chris, ALIC is not good acronym. Decided to drop use of acronym everywhere